tests/new/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -53,13 +53,14 @@ abstract class A
     }
 }
 
-$class = (new Foo())->veryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongMethod();
+$class = (new Foo())->
```

Comment:
Should we handle `new`, `clone` as `$this`?

Reply:
Also feel free create new issue around this problem. PR already is big, let's do it in other PR :+1: